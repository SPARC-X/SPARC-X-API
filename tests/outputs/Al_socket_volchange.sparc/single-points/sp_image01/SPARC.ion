# Ion File Generated by SPARC ASE Calculator
# 
# ASE-SORT:
# 0 1 2 3
# END ASE-SORT

ATOM_TYPE: Al
N_TYPE_ATOM: 4
PSEUDO_POT: 13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
COORD:
0.31002635498075 -0.11676125982981 -0.10080809587731
-0.20478930058667 4.03013608293795 3.42568530055092
4.19798121279424 -0.14528572934667 3.92585490688121
3.81736694579949 4.14402369917815 -0.39320327388543

