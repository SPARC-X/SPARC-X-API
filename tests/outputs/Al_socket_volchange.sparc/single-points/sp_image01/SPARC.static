***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Al:
      0.0401072929       0.9848949034       0.9869587222
      0.9735069476       0.5213680896       0.4431718840
      0.5430817720       0.9812047679       0.5078775085
      0.4938427068       0.5361014305       0.9491323282
Total free energy (Ha): -9.043784594535243E+00
Atomic forces (Ha/Bohr):
 -4.6545659010E-03   1.0910500836E-02  -2.1484091463E-03
  7.9034706129E-03  -1.0332272794E-02   4.6320681206E-03
 -5.5312544787E-03   1.4037844226E-02  -9.0032967529E-03
  2.2823497668E-03  -1.4616072269E-02   6.5196377787E-03
Stress (GPa): 
 -7.3370513709E+00   1.8635028645E+00  -7.2876683300E-01 
  1.8635028645E+00  -2.2951430546E+01   1.6496611617E+00 
 -7.2876683300E-01   1.6496611617E+00  -9.4337661633E-01
