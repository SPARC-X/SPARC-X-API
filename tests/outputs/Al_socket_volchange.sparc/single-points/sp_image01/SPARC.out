***************************************************************************
*                       SPARC (version Oct 31, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Thu Jan 18 08:03:27 2024                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 1 1 1 
LATVEC:
7.729924717735950 0.000000000000000 0.000000000000000 
0.000000000000000 7.729924717735950 0.000000000000000 
0.000000000000000 0.000000000000000 7.729924717735950 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.000950043469
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 12
CHEB_DEGREE: 17
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-03
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-05
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.66E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0
MIXING_PARAMETER: 1
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
NUM_CHEFSI: 1
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: SPARC
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
7.729924717735950 0.000000000000000 0.000000000000000 
0.000000000000000 7.729924717735950 0.000000000000000 
0.000000000000000 0.000000000000000 7.729924717735950 
Volume: 4.6187642213E+02 (Bohr^3)
Density: 2.3366889676E-01 (amu/Bohr^3), 2.6184639958E+00 (g/cc)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 2
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  2
Mesh spacing                       :  0.515328 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  SPARC.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
Atomic mass                        :  26.9815385
Pseudocharge radii of atom type 1  :  7.21 7.21 7.21 (x, y, z dir)
Number of atoms of type 1          :  4
Estimated total memory usage       :  3.01 MB
Estimated memory per processor     :  1.50 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2623630099E+00        1.704E-01        0.019
2            -2.2628749425E+00        2.103E-01        0.007
3            -2.2614515885E+00        1.131E-01        0.007
4            -2.2612206962E+00        9.719E-02        0.006
5            -2.2609937363E+00        4.687E-02        0.006
6            -2.2609525471E+00        1.610E-02        0.006
7            -2.2609562192E+00        1.977E-02        0.006
8            -2.2609477131E+00        2.768E-03        0.006
9            -2.2609461486E+00        5.654E-04        0.006
Total number of SCF: 9     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2609461486E+00 (Ha/atom)
Total free energy                  : -9.0437845945E+00 (Ha)
Band structure energy              :  4.6705173304E-01 (Ha)
Exchange correlation energy        : -4.3409340657E+00 (Ha)
Self and correction energy         : -1.2498725960E+01 (Ha)
-Entropy*kb*T                      : -3.5207108341E-03 (Ha)
Fermi level                        :  1.2062129640E-01 (Ha)
RMS force                          :  1.4899969649E-02 (Ha/Bohr)
Maximum force                      :  1.7570293084E-02 (Ha/Bohr)
Time for force calculation         :  0.014 (sec)
Pressure                           :  1.0410619511E+01 (GPa)
Maximum stress                     :  2.2951430546E+01 (GPa)
Time for stress calculation        :  0.026 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.184 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
